### DISCUSSION TITLE
BirdNET v2.5 model species classes?

### DISCUSSION THREAD

**Discussion by tphakala** [birdnet-team/BirdNET-Analyzer#443] [General]
According to this video (https://www.youtube.com/watch?v=Faavvmi9JZw), the upcoming v2.5 model will add a significant number of non-bird sounds to its detection library. Currently, BirdNET already detects some non-bird species, but there's no clear classification, in labels or any external resource, to indicate which family or category each detected sound belongs to (e.g., bird, amphibian, insect, canine, man-made sound, etc.).

Could the v2.5 model address this issue?

**Comment by kahst** [comment]
We'll use iNatSounds data to add non-birds and what they do is add the taxonomic group to the label name. I think this is what we'll do too which should help to distinguish birds from non-birds. We'll also try to update the geomodel to include the non-birds.

**Comment by conormcmahon** [comment]
I think that would be a really useful addition!
I have a temporary version of this that I set up for my own usage here, in case it's useful to either of you.
https://www.reddit.com/r/BirdNET_Analyzer/comments/14i527d/nonbird_sounds_and_taxonomic_metadata_v_24/
This was a semi-manual process so there might be some mistakes in there. I did something similar to what you're suggesting, but I drew the labels from GBIF.

**Comment by Svardsten53** [comment]
Is there any time estimate for when 2.5 will be released?

Will this be easily included in the original version of BirdNET-Pi just like 2.4 did or will coding be required to make it work?

**Comment by tphakala** [comment]
Has plans for releasing a new model been abandoned?
